Build console metrics exporter failed on Windows with VS2017

Got below error when building console metrics exporter with VS2017.

..\exporters\ostream\src\metric_exporter.cc(36): error C2039: 'strftime': is not a member of 'std'

--- a/exporters/ostream/src/metric_exporter.cc
+++ b/exporters/ostream/src/metric_exporter.cc
@@ -3,8 +3,8 @@
 
 #include "opentelemetry/exporters/ostream/metric_exporter.h"
 #include <algorithm>
-#include <ctime>
 #include <chrono>
+#include <ctime>
 #include <map>
 #include "opentelemetry/exporters/ostream/common_utils.h"
 #include "opentelemetry/sdk/metrics/aggregation/default_aggregation.h"

--- a/exporters/ostream/src/metric_exporter.cc
+++ b/exporters/ostream/src/metric_exporter.cc
@@ -3,6 +3,7 @@
 
 #include "opentelemetry/exporters/ostream/metric_exporter.h"
 #include <algorithm>
+#include <ctime>
 #include <chrono>
 #include <map>
 #include "opentelemetry/exporters/ostream/common_utils.h"

